designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
+
+Exposing Firmware Config for VM.
+Firmware is placed within the scheduling section. It is related to CPU, and shoudn't be seperated from the rest of the CPU features. Because of that, it should be located next to other CPU features. Also, it affects performance, so it is related to "Resource Requirements".
+
+![VM page](img/Firmware-0-0.jpg)
+Firmware Config is placed in the Scheduling sction of the VM details tab.
+The default value of this field is "Default"
+
+![Config modal](img/Firmware-1-0.jpg)
+This modal is composed of three fields:
+- Bootload (Selection): BIOS (Default) and EFI
+- Serial (string): [The system-serial-number in SMBIOS]
+- UUID ( string): [UUID reported by the vmi bios. Defaults to a random generated UID]
+
+![Config modal](img/Firmware-1-1.jpg)
+Changing the Bootload to EFI
+
+![Config modal popovers](img/Firmware-1-2.jpg)
+For each feature, a popover will be available on clicking the field name.
+Bootloader link: https://kubevirt.io/api-reference/v0.26.0/definitions.html#_v1_bootloader
+Serial link: https://kubevirt.io/api-reference/v0.26.0/definitions.html#_v1_firmware
+UUID link: https://kubevirt.io/api-reference/v0.26.0/definitions.html#_v1_firmware
+
+![VM page](img/Firmware-2-0.jpg)
```

Comment:
Everywhere else in the console we still have the pencil next to the value rather than the field label. I think we should be consistent here. @rachael-phillips investigated this a bit more with Gina and since we now have the underlined fields where users can click on the field label to get additional descriptions...moving the pencil there too would make it difficult to differentiate between the underline click radius and the pencil.

Reply:
Updated the edit pencil icon placement